packages/flutter/lib/src/material/colors.dart

```diff
@@ -2,7 +2,76 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:ui' show Color;
+import 'dart:ui' show Color, hashValues;
+
+class _ColorSwatch extends Color {
```

Comment:
It doesn't make sense for this class to be private since it's the superclass of a public class.